dataland-frontend/src/components/forms/parts/fields/NaceCodeFormFieldBindData.vue

```diff
@@ -0,0 +1,61 @@
+<template>
```

Comment:
Also for this component. I don't understand why we cannot just use the `NaceCodeFormField` we already have.
We just want to submit nace codes as a list of strings for a procurement type, so it is the same use case as the one we already have.
Can you explain this?

Reply:
Because we would have to look for a solution to store it state differently than in the parent

Reply:
Yes got it now. Closing it